src/StackHCIVM/StackHCIVM.Autorest/docs/Get-AzStackHCIVMImage.md

```diff
@@ -48,7 +48,7 @@ Get-AzStackHCIVMImage -Name "testimage" -ResourceGroupName "test-rg"
 ```output
 Name            ResourceGroupName
 ----            -----------------
-testImage       test-rg
```

Comment:
Why do you make this change. The original one seems better.

Reply:
updated